CHANGES.rst

```diff
@@ -10,6 +10,188 @@
 
 .. towncrier release notes start
 
+3.9.2 (2024-01-28)
+==================
+
+Bug fixes
+---------
+
+- Fixed websocket connection leak.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7978`.
+
+
+
+- Fixed `web.FileResponse` doing blocking I/O in the event loop.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8012`.
+
+
+
+- Fixed double compress when compression enabled and compressed file exists.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8014`.
+
+
+
+- Added runtime type check for ``ClientSession`` ``timeout`` parameter.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8021`.
+
+
+
+- Fixed an unhandled exception in the Python HTTP parser on header lines starting with a colon -- by :user:`pajod`.
+
+  Invalid request lines with anything but a dot between the HTTP major and minor version are now rejected.
+  Invalid header field names containing question mark or slash are now rejected.
+  Such requests are incompatible with :rfc:`9110#section-5.6.2` and are not known to be of any legitimate use.
+
+  (BACKWARD INCOMPATIBLE)
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8074`.
+
+
+
+- Improved validation of paths for static resources -- by :user:`bdraco`.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8079`.
+
+
+
+
+Features
+--------
+
+- Added support for passing `True` to `ssl` while deprecating `None` -- by :user:`xiangyan99`.
```

Comment:
The single backticks here should be double backticks or explicit roles. I like to make updates to the individual change notes before the release, but updating this file post-generation is also fine. It could look like this:
```suggestion
- Added support for passing :py:data:`True` to ``ssl`` while deprecating :py:data:`None` -- by :user:`xiangyan99`.
```

But then, this wording is still incomplete — it doesn't provide any context on where this is happening and whether it's in client or server mode etc. Can we include that?

Regarding the byline — it can be either at the end of the first sentence or paragraph, at the very end. If you choose the end position, don't add a trailing comma, just let it sit on the last line after one empty line-separator.

Here's some examples of what I mean: https://frozenlist.aio-libs.org/en/latest/changes/#contributor-facing-changes / https://ansible-pylibssh.readthedocs.io/en/latest/contributing/guidelines/#examples-for-changelog-entries-adding-to-your-pull-requests.